geoportal/c2cgeoportal_geoportal/scaffolds/create/geoportal/+package+_geoportal/__init__.py_tmpl

```diff
@@ -16,6 +17,11 @@ def main(global_config, **settings):
     )
 
     config.include('c2cgeoportal_geoportal')
+    config.get_settings()['pyramid.available_languages'] = 'en'
```

Comment:
Should'nt we use available_languages from settings ? I do not see any load of yaml settings here.

Reply:
This is a hack to make c2cgeoform happy ..., It should be remove later ...